features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/org.wso2.carbon.identity.core.server.feature.default.json

```diff
@@ -354,6 +354,9 @@
   "identity_mgt.user_claim_update.mobile.enable_verification": false,
   "identity_mgt.user_claim_update.mobile.verification_sms_otp_validity": "5",
   "identity_mgt.user_claim_update.use_verify_claim": true,
+  "identity_mgt.user_claim_update.uniqueness.enable": false,
```

Comment:
What if we enable this by default?

Reply:
+1

Reply:
Disabling by default as additional overhead when it is not using